__tests__/__e2e__/cli.e2e.js

```diff
@@ -22,23 +22,28 @@ describe('End-to-End CLI', () => {
       })
     } catch (err) {
       expect(err.code).toBe(2) // means that vulnerabilities were found
+      console.log(err)
+      console.log(err && err.stdout)
+      console.log(err && err.stderr)
     }
   })
 
   test('CLI should show vulnerabilities breakdown numbers and their titles', async () => {
-    expect.assertions(8)
+    expect.assertions(9)
```

Comment:
Nice catch!

In general, do you really want to bump this number with every time we add more info to the output?

Reply:
I don't.
I think we can change to just has.assertions() as the generic API